src/main/java/org/spdx/utility/compare/SpdxComparer.java

```diff
@@ -1456,10 +1456,10 @@ private boolean _isExternalDcoumentRefsEqualsNoCheck() {
 	 * @return true if the external document refs are equal
 	 * @throws SpdxCompareException on compare errors
 	 */
-	public boolean isExternalDcoumentRefsEquals() throws SpdxCompareException {
```

Comment:
This ones been around for a while, so we should deprecate the miss-spelled version